app/payment-link-v2/reference/reference.njk

```diff
@@ -0,0 +1,52 @@
+{% extends "../../views/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "../../views/macros/error-summary.njk" import errorSummary %}
+
+{% block pageTitle %}
+  {{ __p('paymentLinksV2.reference.pleaseEnterYour') }} {{product.reference_label}}  - {{ productName }}
+{% endblock %}
+
+{% block contentBody %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {{ govukBackLink({
+      text: "Back",
+      href: backLinkHref
+    }) }}
+
+    {{ errorSummary ({
+      errors: errors,
+      hrefs: {
+        'payment-reference': '#payment-reference'
+      }
+    }) }}
+
+    <form method="post" name="startPaymentForm" class="push-bottom">
```

Comment:
also here the `name` doesn't really apply so could be removed.

Reply:
Good spot - done.
Also removed the `class` attribute as it does not do anything.